server.h

```diff
@@ -39,6 +39,8 @@ struct cg_server {
 #endif
 
 	bool xdg_decoration;
+	int output_rotation;
```

Comment:
Please use the proper `enum wl_output_transform` type here. And perhaps rename `output_rotation` to `output_transform` instead.